_config.yml

```diff
@@ -56,16 +56,6 @@ defaults:
     values:
       image: /assets/images/cards/card-default.png
 
-# Conversion
-markdown: kramdown
-highlighter: rouge
-
-# Markdown Processors
-kramdown:
-  input: GFM
```

Comment:
Kramdown is about the markdown processing, did you double-check that markdown extensions provided by Github (ie [Github Flavored Markdown](https://github.github.com/gfm/)) — like tables — are still available in case we need them?

Reply:
Yes I did and they are.

![2022-09-14-Safari](https://user-images.githubusercontent.com/3286488/190106412-d1f2faf8-6ffb-4b83-811b-e6232a3d8993.png)

But you're right, @dirtyhenry, it might be safer to set a Markdown processor anyway. I just did (with GFM).